homeassistant/components/airzone/__init__.py

```diff
@@ -17,12 +17,14 @@
 from .const import DOMAIN, MANUFACTURER
 from .coordinator import AirzoneUpdateCoordinator
 
-PLATFORMS: list[Platform] = [Platform.SENSOR]
+PLATFORMS: list[Platform] = [Platform.CLIMATE, Platform.SENSOR]
 
 
 class AirzoneEntity(CoordinatorEntity):
     """Define an Airzone entity."""
 
+    coordinator: AirzoneUpdateCoordinator
```

Comment:
I guess I should rebase due to1072aff0172ae7d4127181af3ac2097c2e5c3398 and remove this, but I didn't want to disturb any pending review.
@bdraco since you have self-requested a review of this, should I proceed or should I wait?

Reply:
I haven't had time to look at this, so go right ahead and re-base

Reply:
> I haven't had time to look at this, so go right ahead and re-base

Don't worry, there's no need to hurry. I just didn't want to disturb any on-going review :).
Thanks :)!

Reply:
Done in b52034defc495c737e7333f9470f7a516639a06c